### PR TITLE
June newsletter

### PR DESCRIPTION
**Pull request by GiudGiud**
refs #11447

@idaholab/moose-team need more content

### CODE REVIEW

File: modules/doc/content/newsletter/2023/2023_06.md [hunk 1]
```diff
@@ -93,3 +109,22 @@ TIMPI changes:
 ## PETSc-level Changes
 
 ## Bug Fixes and Minor Enhancements
+
+- FillBetweenSidesetsGenerator and SideSetsFromPointsGenerator are now
```
Comment (joshuahansel) [inline modules/doc/content/newsletter/2023/2023_06.md:113]: Make the classes in this list into links (or at least typewriter font).

File: modules/doc/content/newsletter/2023/2023_06.md [hunk 2]
```diff
@@ -1,10 +1,5 @@
 # MOOSE Newsletter (June 2023)
```
Comment (joshuahansel) [inline modules/doc/content/newsletter/2023/2023_06.md:1]: Need the link to this page!

**Review by joshuahansel — CHANGES_REQUESTED** [review]

File: modules/doc/content/newsletter/2023/2023_06.md (hunk 2 above)
Comment (GiudGiud) [reply modules/doc/content/newsletter/2023/2023_06.md:1]: good catch

**Comment by GiudGiud** [comment]
Roy addressed everything

**Review by joshuahansel — APPROVED** [review]

**Pull request merged**